### PR TITLE
FEATURE: SD-316 Remove FF of optional stacking

### PR DESCRIPTION
**Pull request by nadtan**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2295
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2043
- graphene-tests: https://github.com/gooddata/graphene-tests/pull/3230

# Related Jira tasks

### CODE REVIEW

**Comment by nadtan** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3989/ : SUCCESS in 8m 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5050/ : SUCCESS in 7s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2173/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1519/ : SUCCESS in 10m 45s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5051/ : SUCCESS in 8s

**Comment by nadtan** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1030.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1520/ : SUCCESS in 15m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5052/ : SUCCESS in 15s

File: src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.spec.tsx [hunk 1]
```diff
@@ -47,117 +47,6 @@ describe("PluggableAreaChart", () => {
         document.clear();
     });
 
-    it("should reuse one measure, only one category and one category as stack", async () => {
```
Comment (nadtan) [inline src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.spec.tsx]: Covered in https://github.com/gooddata/gooddata-react-components/pull/1030/files#diff-fe5a846a7ede8c4874d77c3b7706a3ddR236

File: src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.spec.tsx [hunk 2]
```diff
@@ -47,117 +47,6 @@ describe("PluggableAreaChart", () => {
...
-        );
-        expect(extendedReferencePoint).toEqual({
-            buckets: expectedBuckets,
-            filters: expectedFilters,
-            uiConfig: uiConfigMocks.oneMetricAndManyCategoriesAreaUiConfig,
-            properties: {},
-        });
-    });
-
-    it("should reuse all measures, only one category and no stacks", async () => {
```
Comment (nadtan) [inline src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.spec.tsx]: Covered in https://github.com/gooddata/gooddata-react-components/pull/1030/files#diff-fe5a846a7ede8c4874d77c3b7706a3ddR300

File: src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.spec.tsx [hunk 3]
```diff
@@ -47,117 +47,6 @@ describe("PluggableAreaChart", () => {
...
-        );
-        expect(extendedReferencePoint).toEqual({
-            buckets: expectedBuckets,
-            filters: expectedFilters,
-            uiConfig: uiConfigMocks.multipleMetricsAndCategoriesAreaUiConfig,
-            properties: {},
-        });
-    });
-
-    it("should return reference point with Date in categories even it was as second item", async () => {
```
Comment (nadtan) [inline src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.spec.tsx]: Covered in https://github.com/gooddata/gooddata-react-components/pull/1030/files#diff-fe5a846a7ede8c4874d77c3b7706a3ddR332

File: src/internal/components/pluggableVisualizations/columnChart/tests/PluggableColumnChart.spec.tsx [hunk 4]
```diff
@@ -126,34 +125,5 @@ describe("PluggableColumnChart", () => {
             expect(measures).toEqual(["m3", "m4"]);
             expect(axis).toEqual(AXIS.DUAL);
         });
-
-        it("should update supported properties list base on axis type", async () => {
```
Comment (nadtan) [inline src/internal/components/pluggableVisualizations/columnChart/tests/PluggableColumnChart.spec.tsx]: Covered in https://github.com/gooddata/gooddata-react-components/pull/1030/files#diff-38fc01393275f995733e0ce7e6101b46R229

Comment (nadtan) [inline src/internal/components/pluggableVisualizations/columnChart/tests/PluggableColumnChart.spec.tsx]: Covered in https://github.com/gooddata/gooddata-react-components/pull/1030/files#diff-38fc01393275f995733e0ce7e6101b46R229

File: src/internal/components/pluggableVisualizations/barChart/tests/PluggableBarChart.spec.tsx [hunk 5]
```diff
@@ -136,34 +135,5 @@ describe("PluggableBarChart", () => {
             expect(measures).toEqual(["m3", "m4"]);
             expect(axis).toEqual(AXIS.DUAL);
         });
-
-        it("should update supported properties list base on axis type", async () => {
```
Comment (nadtan) [inline src/internal/components/pluggableVisualizations/barChart/tests/PluggableBarChart.spec.tsx]: Covered in https://github.com/gooddata/gooddata-react-components/pull/1030/files#diff-38fc01393275f995733e0ce7e6101b46R229

**Comment by naibinh** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2184/ : SUCCESS in 12m 49s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4009/ : SUCCESS in 6m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1048/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1221/ : SUCCESS in 13m 08s

**Pull request merged**